js_modules/dagit/packages/core/src/instance/BackfillTable.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import {MockedProvider} from '@apollo/client/testing';
+import React from 'react';
+
+import {StorybookProvider} from '../testing/StorybookProvider';
+
+import {BackfillTable} from './BackfillTable';
+import {
+  BackfillTableFragmentCancelledAssetsPartitionSetStatus,
+  BackfillTableFragmentCompletedAssetJobStatus,
+  BackfillTableFragmentCompletedOpJobStatus,
+  BackfillTableFragmentFailedErrorStatus,
+  BackfillTableFragmentRequested2000AssetsPureStatus,
+  BackfillTableFragments,
+} from './BackfillTable.mocks';
+
+// eslint-disable-next-line import/no-default-export
+export default {component: BackfillTable};
+
+export const GeneralStates = () => {
+  return (
+    <StorybookProvider>
```

Comment:
Similar comment to elsewhere -- do we need the full `StorybookProvider`, if we're not using the full mocked Apollo provider?

Reply:
Hmm, it looks like the two elements that this still requires are the `CustomAlertProvider` and the `WorkspaceProvider` (to build some misc URLs in some backfill table states - not super important).

Maybe another option would be exporting a StorybookProvider that doesn't include the apollo part, and then having each page use the version of Apollo mocking it wants?